### PR TITLE
conops template

### PR DESCRIPTION
**Pull request by capsulecorplab**
Draft template for conops of #97 

Preview of conops template available [here](https://github.com/spacedecentral/Coral/blob/97-create-document-templates/docs/CONOPS_TEMPLATE.adoc)

### CODE REVIEW

File: docs/CONOPS_TEMPLATE.adoc [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+
+== Governance
+
+=== Governance policies
+
+_Describe policies or principles which will govern any critical business and technological decisions in implementing the plan._
+
+=== Organization
+
+_Describe the organization unit that is responsible for the governance, and describe policies on organizational structure and human resource in system development._
```
Comment (Suzibianco) [inline docs/CONOPS_TEMPLATE.adoc]: Test comment

File: docs/CONOPS_TEMPLATE.adoc [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+=== Compliance
+
+_Describe policies on regal and regulatory compliance._
+
+
+'''
+
+===== Attribution
+
+This document is adapted from the ISO/IEC/IEEE 29148:2011 _International Standard - Systems and software engineering -- Life cycle processes --Requirements engineering_, available at https://standards.ieee.org/standard/29148-2011.html.
```
Comment (Engineer1119) [inline docs/CONOPS_TEMPLATE.adoc]: great link

File: docs/CONOPS_TEMPLATE.adoc [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+
+== Purpose
+
+_Describe the purpose of this document by describing the current status of the organization, the goal of the long-range strategic plan, and gaps between them._
+
+== Scope
+
+_Describe the scope of this document by specifying the organization's business domains to which it applies._
```
Comment (Engineer1119) [inline docs/CONOPS_TEMPLATE.adoc]: @capsulecorplab this Conops seems much more focused on the people running the operation than the engineering and mechanical operations of the payload itself. This is in contrast to [John Paterson's ConOps](https://docs.google.com/document/d/1-hRz5XKsrLemaTI3zKRS6VFRAF2KNrFSMr4wgYlgxso/edit) which is much more focused on the latter.

Should we add another section that's more narrowly focused on the engineering/mechanical processes, or is that meant to be embedded in the bigger context that's outline here?

Comment (capsulecorplab) [reply docs/CONOPS_TEMPLATE.adoc]: I noticed that too. We can either deviate from the original ISO/IEC/IEEE 29148:2011 spec and rewrite the template to be more engineering focused, or we can try to refactor John's ConOps doc to fit into the bigger context as you suggest. I don't have a strong preference toward either.

Comment (uditkumarsahoo) [reply docs/CONOPS_TEMPLATE.adoc]: John had actually written the OpsCon, where as Sean has focused on Conops. They are slightly different. Check this source for a quick difference: https://plus.google.com/104099794938550497171/posts/D51e89gh4RR

Comment (uditkumarsahoo) [reply docs/CONOPS_TEMPLATE.adoc]: Document type in this case is Mission Concept and ConOps is the document name

Comment (uditkumarsahoo) [reply docs/CONOPS_TEMPLATE.adoc]: the rest of the document seems fine. However, we can start investigating to determine each points mentioned in the table of content

**Review by uditkumarsahoo — APPROVED** [review]
Good to start.

**Pull request merged**